include/deal.II/fe/fe_values.h

```diff
@@ -1512,6 +1512,11 @@ class FEValuesBase : protected FEValuesData<dim,spacedim>,
    * The same holds for the arguments of this function as for the
    * shape_value() function.
    *
+   * @param function_no Number/indicator of the shape function to be evaluated.
+   *
+   * @param point_no Number/indicator of the quadrature point at which function is to be
```

Comment:
`point_no` is the the actual name of the argument (which is `quadrature_point`). Can you change this? Also, just replace `Number/indicator` by `Number`. That's simpler and descriptive enough.